plaidml/core/core.h

```diff
@@ -128,6 +131,39 @@ inline const char* to_string(DType dtype) {
   }
 }
 
+inline DataType to_datatype(DType dtype) {
```

Comment:
Unfortunately we can't expose the `pmlc::util::DataType` to end-users without forcing them to include all of `pmlc`. The deployment model is that users will pull in `pmlc` as a `.so` (a binary distribution, not a source-based one).